app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
```

Comment:
Just a small nitpick, but why don't you flash messages related to message creation?

Eventually won't you want the user to know what is happening with their comment?

If you just skipped the flash messages to save time just ignore this since they aren't really that relevant to the developer as compared to the user.